crates/hive-utils/src/hive/genesis.rs

```diff
@@ -43,6 +43,7 @@ impl HiveGenesisConfig {
 lazy_static! {
     pub static ref KAKAROT_ADDRESSES: FieldElement = FieldElement::from_hex_be("0x9001").unwrap(); // Safe unwrap, 0x9001
     pub static ref BLOCKHASH_REGISTRY_ADDRESS: FieldElement = FieldElement::from_hex_be("0x9002").unwrap(); // Safe unwrap, 0x9002
+    pub static ref DEPLOYER_ACCOUNT_ADDRESS: FieldElement = FieldElement::from_hex_be("0x9003").unwrap(); // Safe unwrap, 0x9003
```

Comment:
can this change in another place and create a desync? or is it fixed forever

Reply:
If we initialize Madara with this config, these contracts will be fixed to these addresses

Reply:
alright